sdk/identity/azure-identity/pom.xml

```diff
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.19.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;current} -->
+      <version>1.19.0-beta.2</version> <!-- {x-version-update;com.azure:azure-core;current} -->
```

Comment:
Why is `azure-identity` using the current version of azure-core instead of dependency version like `azure-core-http-netty` dependency below?

Reply:
I think it's for this release identity needed to use the latest changes in azure-core - I can rebase this PR once identity is released and updated.

Reply:
Pending PR: https://github.com/Azure/azure-sdk-for-java/pull/22903#pullrequestreview-703424078

Reply:
you can merge this PR, I'll rebase mine and change the version to dep.